src/less/_button.less

```diff
@@ -49,32 +45,35 @@
   }
 }
 
-//== Block
 //
+// == Block ==
 //
 
-.btn-block {
-  display:block;
-  width: 100%;
+.component-button {
```

Comment:
This could be moved in to the curly braces above, rather than repeating `.component-button`